ubg/src/mafia/MafiaDay.js

```diff
@@ -101,7 +105,7 @@ function MafiaDay({mafiaKill, doctorSave, usersData,
    * @return {undefined}
    */
   function startNight() {
-    if (dayVote.length === usersData.length) {
+    if (dayVote.length === alive()) {
```

Comment:
I wonder if it would be a better idea to move alive() into startDay and set total players alive as a state, so that total players alive and players can be initialized when the component loads.

Reply:
Yeah I've tried a wide combination of things and it only works when you call alive() right on the spot of when you need it. There seems to be some issue with the useEffects, but I'm not sure what a good workaround is without doing this.